components/sup/src/manager/mod.rs

```diff
@@ -650,12 +741,7 @@ impl Manager {
     fn start_initial_services_from_watcher(&mut self) -> Result<()> {
         for service_event in self.watcher.initial_events()? {
             match service_event {
-                SpecWatcherEvent::AddService(spec) => {
```

Comment:
You may want to read the service specs directory to get a view of what is stopped vs what is running. As @smith pointed out, things can be "loaded" but not be running